ElectronicObserver/Data/KCDatabase.cs

```diff
@@ -206,8 +207,53 @@ private KCDatabase()
 		TsunDbSubmission = new TsunDbSubmissionManager();
 		FleetPreset = new FleetPresetManager();
 		Translation = new DataAndTranslationManager();
+
+#if DEBUG
+		// data needed for loading old event battles via local api loader
+		// the values don't really matter, they just need to exist
```

Comment:
woudn't it be easier to maintain to programmatically call AddWorld with something like for i = 56, while i < last event id; i++
and call AddMap 7 times for each event

Reply:
true